### PR TITLE
Updates to Quest_LV_0100

### PR DESCRIPTION
**Pull request by HelpfulKoreanGirl**
Filled in some of the blank spaces. Edited Evidence of Salvation to Light of Salvation. I don't know why the questline used to say evidence of salvation, but now the new Korean version texts say Light of Salvation. They're not contradictory, but might be better to choose one for continuity?

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9578,14 +9578,14 @@ QUEST_LV_0100_20151001_009573	Rescue the mysterious girl
 QUEST_LV_0100_20151001_009574	{memo X}You have to save the mysterious girl while Pharets are unconscious due to the scent.
 QUEST_LV_0100_20151001_009575	{memo X}You've obtained the stone!
 QUEST_LV_0100_20151001_009576	You couldn't obtain anything!
-QUEST_LV_0100_20151016_009577	
-QUEST_LV_0100_20151016_009578	$You mean you want to enter the Crystal Mine to find the evidence of salvation? {nl}
-QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they check the evidence of salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
-QUEST_LV_0100_20151016_009580	
+QUEST_LV_0100_20151016_009577	Is this stone tablet the Light of Salvation?{n1} It just looks like an ordinary old tablet to me.{n1}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: {n1} -> {nl} 

(2x)

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9578,14 +9578,14 @@ QUEST_LV_0100_20151001_009573	Rescue the mysterious girl
...
+QUEST_LV_0100_20151016_009578	$You mean you want to enter the Crystal Mine to find the Light of Salvation? {nl}
+QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they Search for the Light of Salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Bishop and Knight Commander -> bishop and the Knight Commander.

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -9578,14 +9578,14 @@ QUEST_LV_0100_20151001_009573	Rescue the mysterious girl
...
+QUEST_LV_0100_20151016_009580	Oh, let me give you some helpful information in return for helping our villagers.{nl}The Light of Salvation that the Revelators are looking for is probably in the Closed Area.
 QUEST_LV_0100_20151016_009581	
-QUEST_LV_0100_20151016_009582	
+QUEST_LV_0100_20151016_009582	The Light of Salvation must be that power that I felt.. {n1} At least that's what I believe.
 QUEST_LV_0100_20151016_009583	
-QUEST_LV_0100_20151016_009584	
+QUEST_LV_0100_20151016_009584	Thank you for saving me.{n1}You must also be a Revelator who has come in search of the Light of Salvation.{n1}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: {n1} -> {nl}

(2x)

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -9578,14 +9578,14 @@ QUEST_LV_0100_20151001_009573	Rescue the mysterious girl
...
-QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they check the evidence of salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
-QUEST_LV_0100_20151016_009580	
+QUEST_LV_0100_20151016_009577	Is this stone tablet the Light of Salvation?{n1} It just looks like an ordinary old tablet to me.{n1}
+QUEST_LV_0100_20151016_009578	$You mean you want to enter the Crystal Mine to find the Light of Salvation? {nl}
+QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they Search for the Light of Salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
+QUEST_LV_0100_20151016_009580	Oh, let me give you some helpful information in return for helping our villagers.{nl}The Light of Salvation that the Revelators are looking for is probably in the Closed Area.
 QUEST_LV_0100_20151016_009581	
-QUEST_LV_0100_20151016_009582	
+QUEST_LV_0100_20151016_009582	The Light of Salvation must be that power that I felt.. {n1} At least that's what I believe.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: {n1} -> {nl}

**Comment by Sunaries** [comment]
Hey there,

Thanks for contributing. Nice catch on the "Light of Salvation". :smile: 
By the way, Korean text isn't permitted in the English files, which line 9845, 9877 and 9886 still have. It has to be either blank or translated in English. 

@imcgames "Light of Salvation" is final?

**Comment by imcgames** [comment]
"Light of Salvation" is good :+1:

**Comment by HelpfulKoreanGirl** [comment]
Ah thanks for pointing out the {n1} is {nl}, Sunaries. Sorry about the Korean text! That was inadvertent.

I'd like to help with more contributions, but I'll start with just a few lines at a time until I get a better grasp of the conventions and flow of the text.. and then gradually increase the number.

Thanks for your help!

**Comment by ttgmichael** [comment]
Hey @HelpfulKoreanGirl,

It seems your pull request has some merge conflicts (which means IMC can't merge it unless your repository has some fixes). I opened a pull request on your repository that should have the conflicts fixed, which would enable IMC to pull these changes to the main translation repository.

Let me know if you have any other questions. Join us at Discord as commented by @Sunaries on your other PR :)

**Pull request closed without merging**